Dfe.PrepareConversions/Dfe.PrepareConversions/Areas/Transfers/Pages/Projects/PublicSectorEqualityDuty/Task.cshtml

```diff
@@ -39,7 +39,7 @@
             </div>
         </p>
 
-        <p>Further information on <a class="govuk-link" href="https://educationgovuk.sharepoint.com/sites/design-good-policy/SitePages/psed.aspx?web=1" target="_blank" rel="noopener" data-cy="psed-task-further-info">meeting the Public Sector Equality Duty (opens in a new tab) on in SharePoint</a></p>
+        <p>Further information on <a class="govuk-link" href="https://educationgovuk.sharepoint.com/sites/design-good-policy/SitePages/psed.aspx?web=1" target="_blank" rel="noopener" data-cy="psed-task-further-info">meeting the Public Sector Equality Duty (opens in a new tab)</a> on in SharePoint</p>
```

Comment:
Further information on <a class="govuk-link" href="https://educationgovuk.sharepoint.com/sites/design-good-policy/SitePages/psed.aspx?web=1" target="_blank" rel="noopener" data-cy="psed-task-further-info">meeting the Public Sector Equality Duty (opens in a new tab)</a> **is on** SharePoint